webui/react/src/hooks/useModal/Workspace/useModalWorkspaceCreate.tsx

```diff
@@ -227,8 +227,6 @@ const useModalWorkspaceCreate = ({ onClose, workspaceID }: Props = {}): ModalHoo
           checkpointStorageConfig?: unknown;
           name: string;
         } = {
-          agentUserGroup: {},
```

Comment:
is there doc for this usage? its a bit confusing what to set in variables

Reply:
I think our swagger file is the closest thing to API documents...

Reply:
ah ok. the type i put wasnt actually correct. i'll fix it soon